### ISSUE TITLE
Accessibility (WCAG level A):  No alternative text in the SVG graphs we generate for Statistics

### ISSUE THREAD

**Issue by claycompton** [NuGet/NuGetGallery#1295]
Right now, all of the SVG graphics we generate have no accessible text.  Because of this, no text is reported through MSAA and UIA, which are the two main accessibility technologies and Windows and the two main ways screen readers get their information.

To fix this, we need to add a &lt;title&gt; and &lt;desc&gt; element to each SVG graphic we generate.  Ideally, the content should be like the following:

```
<svg height="480" width="640">
    <title>NuGet Client Usage (Last 6 Weeks)</title>
    <desc>This is a graph showing the number of downloads by each version of the NuGet client over the last six weeks.</desc>
    <g xmlns="http://www.w3.org/2000/svg" transform="translate(80 20)">
    ...
    </g>
</svg>
```
